docs/development.md

```diff
@@ -74,4 +74,4 @@ This project follow [Semantic Versioning](https://semver.org/)
 
 ## Changelog
 
-This project follows [keep a changelog](https://keepachangelog.com/en/1.0.0/) guidelines for changelog.
```

Comment:
What is the change here?

Reply:
There is some empty line added by me by mistake. This can be ignored.